db/migrate/20191015103157_add_fields_to_payments.rb

```diff
@@ -0,0 +1,11 @@
+class AddFieldsToPayments < ActiveRecord::Migration[5.2]
```

Comment:
Ah, great spot, thank you! I did not see this because I must not have had any payrolled claims in my development database when I ran the data migration, and the spec wouldn't catch this since it tests the data migration in isolation.

I've reproduced your issue locally, renamed the data migration to fix the ordering, and confirmed that this fixes the issue.

Reply:
👍